pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.dataone</groupId>
     <artifactId>dataone-index-worker</artifactId>
-    <version>2.4.0</version>
+    <version>DEVELOP</version>
```

Comment:
Any reason you don't want the version like `2.4.0`? If you want to be the develop version, I believe the convention of version is `2.4.0-SNAPSHOT`

Reply:
+1 on the SNAPSHOT convention. I had left it at 2.4.0 to make image management easier but its fine to switch to SNAPSHOT. This is likely to be 3.0.0-SNAPSHOT though, right?

Reply:
done